# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/base/content/test/contextMenu/browser.toml
# Commit: 32c74a4cd05b
# Full Hash: 32c74a4cd05b13504875d6c230bf57107181ce5f
# Author: abhishekmadan30 <148475204+abhishekmadan30@users.noreply.github.com>
# Date: 2024-03-26 04:11:24
# Regressor Bug: 1877419
# File Overlap Count: 1
# Description:
#   Bug 1877419 - Part 2: Added tests for nested stripping for Strip on Share. r=pbz
#   
#   Depends on D202943
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D203082
# ==============================================================================

diff -r c2ca2f596c09 -r 32c74a4cd05b browser/base/content/test/contextMenu/browser.toml
--- a/browser/base/content/test/contextMenu/browser.toml	Mon Mar 25 18:30:38 2024 +0000
+++ b/browser/base/content/test/contextMenu/browser.toml	Mon Mar 25 18:30:39 2024 +0000
@@ -86,6 +86,8 @@
 
 ["browser_strip_on_share_link.js"]
 
+["browser_strip_on_share_nested_link.js"]
+
 ["browser_utilityOverlay.js"]
 https_first_disabled = true
 skip-if = ["os == 'linux' && socketprocess_networking"]